src/modules/bank/submitBank.js

```diff
@@ -0,0 +1,96 @@
+import addCommas from '../system/addCommas';
+import getElementById from '../common/getElementById';
+import getText from '../common/getText';
+import intValue from '../system/intValue';
+import isUndefined from '../common/isUndefined';
+import querySelector from '../common/querySelector';
+import querySelectorArray from '../common/querySelectorArray';
+import { clearError, showError } from './error';
+
+function getDepositAmount() {
+  const input = getElementById('deposit_amount');
+  return Number(input.value);
+}
+
+function getWithdrawAmount() {
+  const input = getElementById('withdraw_amount');
+  return Number(input.value);
+}
```

Comment:
getDepositAmount and getWithdrawAmount are almost identical, could be refactored.
getElementById('deposit_amount') and getElementById('withdraw_amount') exist twice.